### DISCUSSION TITLE
First release

### DISCUSSION THREAD

**Discussion by notreux** [NotReux/UpsideEngine#3] [Announcements]
Here is the first release of the engine, this release groups the versions from 1.0.0 to 1.0.2, this happens because we should have started using the github releases before, but don't worry, from now on we will use them as we should.

<hr /><em>This discussion was created from the release <a href='https://github.com/TheHackerPuppy/UpsideEngine/releases/tag/v1.0.0'>First release</a>.</em>

**Comment by ewiborn** [comment]
**Good luck**

**Comment by alxcg121** [comment]
Congrats with your first official release, and good luck with the project!
